src/many/src/server/module/_9_account/features/multisig.rs

```diff
@@ -184,13 +188,13 @@ pub struct ApproverInfo {
 #[cbor(map)]
 pub struct InfoReturn {
     #[n(0)]
-    pub memo: Option<ByteVec>,
+    pub memo: Option<String>,
```

Comment:
Either fix the spec to make it optional or fix it here to make it mandatory

Reply:
This is optional.